static/js/pm_list.js

```diff
@@ -1,51 +1,91 @@
 import $ from "jquery";
+import _ from "lodash";
+
+import render_more_private_conversations from "../templates/more_pms.hbs";
 
-import * as narrow_state from "./narrow_state";
 import * as pm_list_data from "./pm_list_data";
 import * as pm_list_dom from "./pm_list_dom";
+import * as resize from "./resize";
 import * as stream_popover from "./stream_popover";
 import * as ui from "./ui";
 import * as ui_util from "./ui_util";
 import * as vdom from "./vdom";
 
 let prior_dom;
-let private_messages_open = false;
 
 // This module manages the "Private messages" section in the upper
 // left corner of the app.  This was split out from stream_list.js.
 
-function get_filter_li() {
-    return $(".top_left_private_messages .private_messages_header");
+let private_messages_collapsed = false;
+
+// To keep a track whether we're zoomed in or out.
```

Comment:
"To keep track of whether we're zoomed in or out", no "a"

Reply:
I'd actually make this more specific "The private messages section can be zoomed into view more messages. This keeps track of if we're zoomed in or not."

Reply:
Updated the PR with this, thanks :)

Reply:
Updated the PR with this, Thanks @evykassirer for the comment.